composable/assetlist.json

```diff
@@ -229,6 +229,350 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/tinkernet/images/tnkr.svg"
       }
+    },
+    {
+      "description": "The native token of Ethereum, bridged via IBC.",
+      "denom_units": [
+        {
+          "denom": "eth-wei",
+          "exponent": 0
+        },
+        {
+          "denom": "eth",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/4C14B42547F6428A4E84F501D21CE5DF6C40C537635A8B0E6A5ED8FB4919BCE7",
+      "name": "Ethereum",
+      "display": "eth",
+      "symbol": "ETH",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "wei",
+            "channel_id": ""
+          },
+          "chain": {
+            "channel_id": "",
```

Comment:
Please register the ibc connection to the /IBC/ directory, then fill in the `channel_id`s and `path` here (for each new asset) to match.